.changeset/slow-hounds-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'layerchart': patch
+---
+
+Exposed `ForceSimulation`'s default values by exporting them as constants.
```

Comment:
Forgot to mention, but I try to follow [conventional commits](https://www.conventionalcommits.org/) for changesets to easily identify fix, feature, or breaking change and affected component (if applicable.

Could you change this to:
```
fix(ForceSimulation): Expose default values by exporting them as constants
```

Reply:
Fixed. Noted and will try to keep this in mind for future PRs. 👍